.github/ISSUE_TEMPLATE/3_internal_decision.yml

```diff
@@ -0,0 +1,59 @@
+name: Internal - Decision
+description: Describe a decision that needs to be made and recorded
+title: "[ADR] "
```

Comment:
There's a label for "docs:adr" that we could apply and not use the [ADR] prefix.

Reply:
Good idea! Just added that default label in this commit: [feat: Adds "docs: adr" label](https://github.com/HHS/simpler-grants-gov/pull/2369/commits/a9ee8bc55ceb4f547ead6a23f06327c20941b4fb) after dropping the prefix in this commit: [refactor: Drops all title pre-fixes](https://github.com/HHS/simpler-grants-gov/pull/2369/commits/ee23fa1a73749925203c3351a58a0659a98cdc33)